olive/engine/packaging/packaging_config.py

```diff
@@ -36,6 +37,11 @@ class AzureMLModelsPackagingConfig(CommonPackagingConfig):
     description: Optional[str] = None
 
 
+class DockerfilePackagingConfig(CommonPackagingConfig):
+    base_image: str = "mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.6-cudnn8-ubuntu20.04"
```

Comment:
Please make this string a constant in olive/constants.py

Reply:
Will do.